.nvmrc

```diff
@@ -1 +1 @@
-16
+18
```

Comment:
(If pinned to `^18.15.0`, rc should point to `18.15`, too.)

Reply:
We can't defined ^18.15 in the nvmrc, so 18 is sufficient, since it will use the latest v18 release, which matches ^18.15 as the minor is >= 15.